### DISCUSSION TITLE
Setting focus on CtkOptionmenu using configure method

### DISCUSSION THREAD

**Discussion by Stimu-li** [TomSchimansky/CustomTkinter#1754] [Q&A]
I'm currently building something out of customtkinter and there are multiple optionmenus and entry fields in a frame. I'm trying to navigate between those widgets using the tab key. However, it looks like optionmenu widgets don't receive focus and when i researched, i found out that we have to set the focus in optionmenu manually using optionmenu1.configure(takefocus=1) 

However, this does not work for me and it says takefocus isn't a supported argument. What else xan i use?

**Comment by Akascape** [comment]
@Stimu-li Have yout tried `optionmenu.focus_set()`?
